app/models/maintenance_tasks/run.rb

```diff
@@ -448,6 +450,15 @@ def task
 
     private
 
+    def instrument_status_change
+      return unless status && status != previous_changes["status"].first
+      return if status[/ing\z/] # pausing vs paused, cancelling vs cancelled
```

Comment:
Not opinionated much about this - would also skip `running` - just wondering if it's worth to exclude them here rather than filtering these out if needed when subscribing to them

Reply:
I think it could get confusing as to the difference between `running` and `enqueued` (running can happen multiple times after each interruption if I'm not mistaken). I'd rather remove them for now and let someone with a valid use case add them back. IMO the effort to add them back here is less than the sum of code changes required each time someone gets confused by these states. That being said, I can be explicit about which are ignored rather than using a regex, will fix 🙌